app/bundles/UserBundle/Config/config.php

```diff
@@ -148,6 +152,13 @@
                     'router',
                 ],
             ],
+            'mautic.security.saml.helper' => [
+                'class'     => Mautic\UserBundle\Security\SAML\Helper::class,
+                'arguments' => [
+                    'mautic.helper.core_parameters',
+                    'request_stack',
+                ],
+            ],
```

Comment:
```suggestion
```
Let's use Symfony's autowiring. We need to add 
```php
    $services->set(Mautic\UserBundle\Security\SAML\Helper::class);
```
in https://github.com/mautic/mautic/blob/b4264c717ce31fbafafcefc04b02ecb9fb911e62/app/bundles/UserBundle/Config/services.php#L49
as `Security` folder is part of [the default excludes](https://github.com/mautic/mautic/blob/edb485d7b7c326cd245a6dfe00aa94c9416af97d/app/bundles/CoreBundle/DependencyInjection/MauticCoreExtension.php#L12).